XiEditor/AppDelegate.swift

```diff
@@ -79,7 +79,7 @@ class AppDelegate: NSObject, NSApplicationDelegate {
             if let obj = params as? [String : AnyObject], let msg = obj["msg"] as? String {
                 dispatch_async(dispatch_get_main_queue(), {
                     let alert =  NSAlert.init()
-                    alert.alertStyle = .InformationalAlertStyle
+                    alert.alertStyle = .Informational
```

Comment:
This breaks XCode 7.3.1 (which I'm still stuck on for a while). Can the change be made conditional on XCode version?

Other than that, it seems to work fine on 7.3.1. Many thanks!